.docker/scripts/ckan_cli

```diff
@@ -57,17 +57,17 @@ else
 fi
 
 if [ "$COMMAND" = "ckan" ]; then
-    echo "Using 'ckan' command from $ENV_DIR with config ${CKAN_INI}..." >&2
     # adjust args to match ckan expectations
     COMMAND=$(echo "$1" | sed -e 's/create-test-data/seed/')
     shift
+    echo "Using 'ckan $COMMAND $@' command from $ENV_DIR with config ${CKAN_INI}..." >&2
```

Comment:
better to set the command as an arg, echo it, then eval it.
https://linuxhint.com/bash_eval_command/

Reply:
Tried that, it didn't work out. There seems to be some complexity around syntax for passing arguments.